pkg/kubectl/metricsutil/metrics_printer.go

```diff
@@ -65,12 +67,31 @@ func (printer *TopCmdPrinter) PrintNodeMetrics(metrics []metrics_api.NodeMetrics
 		printMetricsLine(w, &ResourceMetricsInfo{
 			Name:      m.Name,
 			Metrics:   m.Usage,
+			Available: convertResourceList(availableResources[m.Name]),
 			Timestamp: m.Timestamp.Time.Format(time.RFC1123Z),
 		})
 	}
 	return nil
 }
 
+// TODO: remove after Heapster API is finished
+func convertResourceList(resourceList api.ResourceList) v1.ResourceList {
```

Comment:
True! Sorry, I was tied to the metrics api which doesn't have this yet, but this is from kube api.